kernel/kernel-api/src/main/java/io/delta/kernel/internal/DeltaLogActionUtils.java

```diff
@@ -219,7 +219,7 @@ public static CloseableIterator<FileStatus> listDeltaLogFilesAsIter(
         endVersionOpt);
 
     // Must be final to be used in lambda
-    final AtomicBoolean hasReturnedAnElement = new AtomicBoolean(false);
+    final AtomicBoolean hasReturnedlogOrCheckPoint = new AtomicBoolean(false);
```

Comment:
`hasReturnedCommitOrCheckpoint`

Reply:
You can add a comment here. Something like: `This variable is used to help determine if we should throw an error if the table history is not reconstructable. Only commit and checkpoint files are applicable.`